charmcraft.yaml

```diff
@@ -1,43 +1,44 @@
 # This file configures Charmcraft.
 # See https://juju.is/docs/sdk/charmcraft-config for guidance.
 
-# (Required)
 name: datahub-k8s-operator
```

Comment:
I suggest `datahub-k8s` for the name, so that it can be deployed as `juju deploy datahub-k8s` rather than `datahub-k8s-operator`.

Reply:
Updated.